sessionctx/variable/sysvar.go

```diff
@@ -311,15 +311,15 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "div_precision_increment", "4"},
 	{ScopeGlobal, "innodb_lru_scan_depth", "1024"},
 	{ScopeGlobal, "innodb_purge_rseg_truncate_frequency", ""},
-	{ScopeGlobal | ScopeSession, "sql_auto_is_null", "OFF"},
-	{ScopeNone, "innodb_api_enable_binlog", "OFF"},
+	{ScopeGlobal | ScopeSession, SQLAutoIsNull, "0"},
+	{ScopeNone, "innodb_api_enable_binlog", "0"},
 	{ScopeGlobal | ScopeSession, "innodb_ft_user_stopword_table", ""},
 	{ScopeNone, "server_id_bits", "32"},
 	{ScopeGlobal, "innodb_log_checksum_algorithm", ""},
-	{ScopeNone, "innodb_buffer_pool_load_at_startup", "OFF"},
+	{ScopeNone, "innodb_buffer_pool_load_at_startup", "1"},
```

Comment:
"0"?

Reply:
The default value is `ON` since MySQL 5.7.7. See : https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_buffer_pool_load_at_startup